### PR TITLE
Implement Mod List as grids and move metadata to the mod page

### LINKED ISSUE #3: Improve long mods list
Not sure how well the new UI would support it, but would be nice to be able to be able to quickly jump to a mod/setting I want to change

A search would be optimal with even setting specific jump links, but even like a simple more compact list of mod names only could possibly be implemented easily?

### PR DESCRIPTION
**Pull request by Banane9**
Closes #3

### CODE REVIEW

**Comment by Nytra** [comment]
Opening MonkeyLoader settings doesn't seem to work.

**Comment by Nytra** [comment]
Otherwise looks good, but there might be too many buttons per row, some of the text looks a bit small and difficult to read.

Screenshot 2024-05-09 012837

File: MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs [hunk 1]
```diff
@@ -232,68 +281,61 @@ await foreach (var feedItem in EnumerateMonkeysAsync(path, mod, EarlyMonkeys))
...
             }
+
+            var monkeyLoaderGroup = new DataFeedGroup();
+            monkeyLoaderGroup.InitBase("MonkeyLoader", path, null, Mod.GetLocaleString("MonkeyLoader.Name"));
+            yield return monkeyLoaderGroup;
+
+            var monkeyLoaderGrouping = new[] { Mod.Loader.Id };
+
+            var openMonkeyLoaderSettings = new DataFeedCategory();
+            openMonkeyLoaderSettings.InitBase("MonkeyLoader.OpenMonkeyLoader", path, monkeyLoaderGrouping, Mod.GetLocaleString("OpenMonkeyLoader.Name"), Mod.GetLocaleString("OpenMonkeyLoader.Description"));
```
Comment (Nytra) [inline MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs:325]: Opening monkey loader settings doesn't work.

File: MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs [hunk 2]
```diff
@@ -232,68 +281,61 @@ await foreach (var feedItem in EnumerateMonkeysAsync(path, mod, EarlyMonkeys))
 
         private static async IAsyncEnumerable<DataFeedItem> EnumerateModsAsync(IReadOnlyList<string> path)
         {
-            var monkeyLoaderCategory = new DataFeedCategory();
-            monkeyLoaderCategory.InitBase("MonkeyLoader", path, null, Mod.GetLocaleString("OpenMonkeyLoader.Name"), Mod.GetLocaleString("OpenMonkeyLoader.Description"));
-            yield return monkeyLoaderCategory;
+            var modsGroup = new DataFeedGroup();
+            modsGroup.InitBase("Mods.Group", path, null, Mod.GetLocaleString("Mods"));
```
Comment (Nytra) [inline MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs:285]: If possible maybe reduce the number of buttons per row?

**Review by Nytra — CHANGES_REQUESTED** [review]

File: MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs (hunk 1 above)
Comment (Banane9) [reply MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs:325]: Fixed

File: MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs (hunk 2 above)
Comment (Banane9) [reply MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs:285]: Doesn't appear to be possible without fuckery (maybe looking for the grid component and changing that?)

**Review by Nytra — APPROVED** [review]

**Pull request merged**